### PR TITLE
Split earning mechanisms

### PR DESCRIPTION
**Pull request by tshuck**
@hatchloyalty/core 

Splits the earning mechanism guides to separate pages based on their associated `engagement_type`.  Adds a guide for awards.  Cleans up links between earning mechanisms, awards, tutorials.

### CODE REVIEW

File: content/static-pages/guides/offers/earning-mechanisms/awards.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
```
Comment (tshuck) [inline content/static-pages/guides/offers/earning-mechanisms/awards.md]: All new.

File: content/static-pages/guides/offers/earning-mechanisms/behavioral.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+title: "Behavioral Earning Mechanisms"
+layout: "tutorial"
+---
+
+# What is a Behavioral Earning Mechanism?
+
+Behavioral Earning Mechanisms enable a loyalty program to target non-transactional behaviors that a marketing initiative may wish to engender in their members. Examples of such marketing initiatives may include capturing interactions outside of Hatch (e.g., filling out a survey, or playing a game of chance to earn a variable award), properties of the member such as their account creation or confirmation, or the number of times a member visits your store.
```
Comment (tshuck) [inline content/static-pages/guides/offers/earning-mechanisms/behavioral.md]: New copy.  Rest of file is old.

File: content/static-pages/guides/offers/earning-mechanisms/externally-fulfillable.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Externally Fulfillable Earning Mechanisms"
+layout: "tutorial"
+---
+
+# What is a Externally Fulfillable Earning Mechanism?
+
+Externally Fulfillable Earning Mechanisms enable a loyalty program to execute marketing initiatives that are fulfilled or verified outside of Hatch. Examples of such marketing initiatives may include providing product discount at the point-of-sale (either as a fulfillable offer, or in exchange for a previously-granted voucher via a [Vouchered Offer](/guides/offers/earning-mechanisms/vouchered)), or other types of externally-verified activities.
```
Comment (tshuck) [inline content/static-pages/guides/offers/earning-mechanisms/externally-fulfillable.md]: New copy.  Rest of file is old.

File: content/static-pages/guides/offers/earning-mechanisms/purchasable.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+---
```
Comment (tshuck) [inline content/static-pages/guides/offers/earning-mechanisms/purchasable.md]: All new.

File: content/static-pages/guides/offers/earning-mechanisms/transactional.md [hunk 5]
```diff
@@ -0,0 +1,465 @@
+---
+title: "Transactional Earning Mechanisms"
+layout: "tutorial"
+---
+
+# What is a Transactional Earning Mechanism?
+
+Transactional Earning Mechanisms enable a loyalty program to target transaction-based behaviors that a marketing initiative may wish to engender in their members. Examples of such marketing initiatives may include incentivizing the purchase of a particular product, or to spend a certain number of dollars in-store within a given time frame.
```
Comment (tshuck) [inline content/static-pages/guides/offers/earning-mechanisms/transactional.md]: New copy.  Rest of file is old.

File: content/static-pages/guides/offers/earning-mechanisms/vouchered.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (tshuck) [inline content/static-pages/guides/offers/earning-mechanisms/vouchered.md]: All new.

File: content/static-pages/guides/offers/earning-mechanisms/awards.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+title: "Earning Mechanism Awards"
+layout: "tutorial"
+---
+
+# What is an Award?
+
+An Award represents the portion of an Earning Mechanism that describes the incentive that a member expects to receive in exchange for engaging with the marketing initiative. Many marketing initiatives aim to change a member's habitual behavior or engage in a behavior that is entirely new. These behavioral changes require an investment on the member's part, and so an appropriate incentive must be considered; for some marketing initiatives and some members, an award of points (that may later be spent on a [purchasable offer](/tutorials/purchasable-offer)) is sufficient incentive; others may require the direct receipt of a valuable [voucher](/tutorials/voucher-award); still other initiatives may grant an award outside of Hatch, such as a product discount.
```
Comment (hatch-carl) [inline content/static-pages/guides/offers/earning-mechanisms/awards.md]: Don't semicolon, just split into sentences.

**Pull request merged**